compute_tools/src/spec.rs

```diff
@@ -244,3 +244,21 @@ pub fn handle_databases(spec: &ClusterSpec, client: &mut Client) -> Result<()> {
 
     Ok(())
 }
+
+pub fn create_writablity_check_data(client: &mut Client) -> Result<()> {
+    let query = "
+    CREATE TABLE IF NOT EXISTS health_check (
+        id serial primary key,
+        updated_at timestamptz default now()
+    );
```

Comment:
Here we still should use `if not exists` because we can't distinguish creation and start as I understand.

Reply:
Yeah, it seems so